ranger-plugin/service/src/main/java/org/apache/hawq/ranger/authorization/Utils.java

```diff
@@ -68,6 +77,30 @@ public static String getVersion() {
     }
 
     /**
+     * Retrieves the authentication
+     * @return kerberos or simple[default]
+     */
+    public static String getAuth() {
+        return System.getProperty(AUTH_KEY_ENV, properties.getProperty(AUTH_KEY_FILE, "simple"));
```

Comment:
public static final String SIMPLE="simple"; and use SIMPLE here. So does EMPTY_STRING for ""